workflow/sync/mutex_test.go

```diff
@@ -126,6 +127,7 @@ func TestMutexLock(t *testing.T) {
 		assert.Len(t, concurrenyMgr.syncLockMap, 1)
 	})
 	t.Run("WfLevelMutexAcquireAndRelease", func(t *testing.T) {
+		os.Setenv("HOLDER_KEY_VERSION", "v1")
```

Comment:
```suggestion
		t.Setenv("HOLDER_KEY_VERSION", "v1")
```

Per #13036

Reply:
Also the env var needs to be documented on the "Environment Variables" page of the docs